scrapers/dc-onlyfans.py

```diff
@@ -9,7 +9,7 @@
     import py_common.graphql as graphql
     import py_common.log as log
 except ModuleNotFoundError:
-    print("You need to download the folder 'py_common' from the community repo! (CommunityScrapers/tree/master/scrapers/py_common)", file=sys.stderr)
+    log.error("You need to download the folder 'py_common' from the community repo! (CommunityScrapers/tree/master/scrapers/py_common)")
```

Comment:
If the module is not found then no `log` will be available.
The `print` needs to stay here

Reply:
ahh yeah, fair enough